dpdk/vr_dpdk_knidev.c

```diff
@@ -324,6 +324,8 @@ vr_dpdk_knidev_init(struct vr_interface *vif)
         else
             /* ...so we use os_idx instead */
             port_id = vif->vif_os_idx;
+    } else if (vif->vif_type == VIF_TYPE_MONITORING) {
+            port_id = 0; /* we always use DPDK port 0 */
```

Comment:
DPDK numerates all the detected Ethernet devices starting from 0. So we might get into an issue if we have no eth devices at all or we have few eth ports and don't what to use the first one.

We can add a loop to look up for the first available eth device or pass eth dev index as a parameter?